source/Octopus.Tentacle.Client/Scripts/Models/Builders/ExecuteKubernetesScriptCommandBuilder.cs

```diff
@@ -6,18 +6,25 @@ namespace Octopus.Tentacle.Client.Scripts.Models.Builders
     public class ExecuteKubernetesScriptCommandBuilder : ExecuteScriptCommandBuilder
     {
         KubernetesImageConfiguration? configuration;
+        string? scriptPodServiceAccountName;
 
         public ExecuteKubernetesScriptCommandBuilder(string taskId)
             : base(taskId, ScriptIsolationLevel.NoIsolation) //Kubernetes Agents don't need isolation since the scripts won't clash with each other (it won't clash more than Workers anyway)
         {
         }
 
-        public ExecuteKubernetesScriptCommandBuilder SetKubernetesImageConfiguration(KubernetesImageConfiguration configuration)
+        public ExecuteKubernetesScriptCommandBuilder WithKubernetesImageConfiguration(KubernetesImageConfiguration configuration)
```

Comment:
There was some feedback from @sburmanoctopus to change all these `SetX` to `WithX` for consistency. I'll do the remainder of this builder in another PR